compute/client.go

```diff
@@ -133,6 +147,47 @@ func (c *Client) executeRequest(method, path string, body interface{}) (*http.Re
 	return nil, oracleErr
 }
 
+// Allow retrying the request until it either returns no error,
+// or we exceed the number of max retries
+func (c *Client) retryRequest(req *http.Request) (*http.Response, error) {
+	// Double check maxRetries is not nil
+	var retries int
+	if c.maxRetries == nil {
```

Comment:
Isn't this caught here https://github.com/hashicorp/go-oracle-terraform/blob/ced23cdc14942f44cdc8bdc8ea9dea3f0569a048/compute/client.go#L65? Or are we doing it as a final check before moving forward?